app/models/non_hmis_assessment.rb

```diff
@@ -113,8 +119,15 @@ def pathways_v3?
     type.include?('PathwaysVersionThree')
   end
 
+  def pathways_v4?
+    type.include?('PathwaysVersionFour')
+  end
+
   def total_days_homeless_in_the_last_three_years
-    if pathways_v3?
+    if pathways_v4?
+      total = total_homeless_nights_sheltered + total_homeless_nights_unsheltered
+      total > 1096 ? 1096 : total
```

Comment:
checkout [`clamp`](https://apidock.com/ruby/Comparable/clamp) I think that'll be a bit more readable.
```suggestion
      total.clamp(0, 1096)
```

Reply:
This does look much cleaner!